cmd/sqlcmd/sqlcmd_test.go

```diff
@@ -90,15 +74,23 @@ func TestValidCommandLineToArgsConversion(t *testing.T) {
 		{[]string{"--version"}, func(args SQLCmdArguments) bool {
 			return args.Version
 		}},
-		{[]string{"-s", "|", "-w", "10", "-W"}, func(args SQLCmdArguments) bool {
-			return args.TrimSpaces && args.ColumnSeparator == "|" && *args.ScreenWidth == 10
-		}},
 	}
 
 	for _, test := range commands {
 		arguments := &SQLCmdArguments{}
-		parser := newKong(t, arguments)
-		_, err := parser.Parse(test.commandLine)
+		cmd := &cobra.Command{
+			Use:   "testCommand",
+			Short: "A brief description of my command",
+			Long:  "A long description of my command",
+			Run: func(cmd *cobra.Command, argss []string) {
+				// Command logic goes here
+				vars := sqlcmd.InitializeVariables(!args.DisableCmdAndWarn)
```

Comment:
>vars := sqlcmd.InitializeVariables(!args.DisableCmdAndWarn) [](http://example.com/codeflow?start=4&length=59)

the kong version of this test never called InitializeVariables or setVars. 
The test is just meant to validate that the command line parsing populates the struct correctly.